staff/francisco-fernandez/cv-styled/styles.css

```diff
@@ -0,0 +1,115 @@
+header{
```

Comment:
You need some general styles for the page like font-family, font-size, color, etc...

Reply:
Yes, you are right... I´ve done